torontoadvotech/client
# Ws 05

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Adds single mentor page view

## Related Issue

(WS-05)[https://torontoadvotech.atlassian.net/browse/WS-5] -->

## Acceptance Criteria

You should be able to view a separate page for each individual mentor

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

N/A

### After

![mentor-page](https://user-images.githubusercontent.com/38121935/115314248-d21d9680-a142-11eb-97b7-e8cd7f315fb7.png)

## Testing Steps / QA Criteria

Visit /mentors route and click on any of the of the single mentors to view the mentor page

Confirm mentor page matches FIGMA minus the bio and skills which will be added in a separate story
